resources/label-definitions.properties

```
@@ -708,7 +708,8 @@ pegdown-formatter=ui
 pending-changes=scm-related
 people-redirector=external
 percentage-du-node-column=listview-column
-perfectomobile=external ios android
+perfectomobile=deprecated
+perfecto=ios android tunnel external
```

```suggestion
perfecto=ios android external
```

`tunnel` is new and also not in https://github.com/jenkins-infra/update-center2/blob/master/resources/allowed-github-topics.properties

Let's keep that conversation separate from the multiple things that this PR is already doing.